as/cNFT/assembly/core.ts

```diff
@@ -35,16 +36,42 @@ export function nft_transfer(token_id: TokenId, bidder_id: AccountId): void {
     persistent_tokens.add(token_id, token, bidder_id)
 
     /* Deleting token from previous owner */
-    persistent_tokens.remove(token.id, token.prev_owner_id)
+    persistent_tokens.remove_from_account(token.id, token.prev_owner_id)
+
 
-    
     // Immiting log event
     const transfer_log = new NftTransferLog()
-    
+
     transfer_log.old_owner_id = token.prev_owner_id
     transfer_log.new_owner_id = token.owner_id
     transfer_log.token_ids = [token.id]
 
     const log = new NftEventLogData<NftTransferLog>('nft_transfer', [transfer_log])
     logging.log(log)
+
+}
+
+
+@nearBindgen
+export function burn_design(token_id: TokenId): void {
```

Comment:
Looks fine, just make sure it's doing same actions as: https://github.com/curaOS/contracts/blob/d5a9a3e358fcce98256104273d70e3406ef9afea/as/NFT/assembly/index.ts#L87

Reply:
Not relevant